### DISCUSSION TITLE
Advanced Jukebox upgrade adds storage for discs

### DISCUSSION THREAD

**Discussion by Wakatsu** [P3pp3rF1y/SophisticatedBackpacks#295] [Ideas]
A little QoL idea of mine, would it be possible to add a small inventory to Jukebox upgrade where only discs could go? I know I lost my music collection more than once due to discs get mixed up with other loot in the backpack, so having them sit in little corner of their own would be much appreciated feature. Not to mention it would make reloading the jukebox much easier and heck, maybe would even allow for a bit of auto in the future?

**Comment by lele92007** [comment]
reupping that and adding my ideas
maybe allow looping and autoplaying by pulling disks from the storage
